codegen/smithy-go-codegen/src/main/java/software/amazon/smithy/go/codegen/GoDependency.java

```diff
@@ -54,6 +54,8 @@
             "github.com/awslabs/smithy-go/httpbinding", null, Versions.SMITHY_GO),
     SMITHY_JSON("dependency", "github.com/awslabs/smithy-go",
             "github.com/awslabs/smithy-go/json", "smithyjson", Versions.SMITHY_GO),
+    SMITHY_IO("dependency", "github.com/awslabs/smithy-go",
```

Comment:
I don't see this being used, but usage might be in another PR. Just want to call out

Reply:
This is used by the aws-sdk-go-v2 pr that i've not posted yet.